libdnf/sack/query.cpp

```diff
@@ -1046,6 +1047,33 @@ Query::Impl::filterPkg(const Filter & f, Map *m)
     map_init_clone(m, dnf_packageset_get_map(f.getMatches()[0].pset));
 }
 
+void
+Query::Impl::filterDepSolvable(const Filter & f, Map *m)
+{
+    assert(f.getMatchType() == _HY_PKG);
+    assert(f.getMatches().size() == 1);
+
+    dnf_sack_make_provides_ready(sack);
+    Pool *pool = dnf_sack_get_pool(sack);
+    Id rco_key = reldep_keyname2id(f.getKeyname());
+
+    Queue out;
```

Comment:
Queue could be replaced by IdQueue

Reply:
@lukash  Please Queue could be replaced by IdQueue

Reply:
I see, thanks, I've missed this wrapper. I'll use it.